rfcs/text/0000-Managed-Access-Data.md

```diff
@@ -0,0 +1,346 @@
+
+This list is meant to get the flavor of what will be required by the
+regulations for NIST 800-53. The Access Control control family of checks
+exist inside a much larger framework of controls made up by the rest of
+NIST 800-53, and it should be noted that completing the other controls
+in the document would be required for certification of the DCP to FISMA
+moderate levels so that it can store controlled access data generated by
+funds from NIH grants.
+
+It should be noted again that though these controls are specific to
+access control requirements in the USA, they are expected to represent a
+good starting point for other controls from other nations.
+
+### **Unresolved Questions**
+
+### Design questions which should be resolved during the RFC process
+
+-   Are search results filtered by access control?
+
+-   (related) Is Metadata always searchable
+
+-   What certifications are required to meet the feature request
+    > "support controlled data in the DCP"? FISMA moderate? GDPR? What
+    > else?
+
+-   Should we have the researchers run the DAC? Perhaps providing them
+    > with a template?
+
+### **Prior Art** 
+
+[[Analysis of five years of controlled access and data sharing
+compliance at the International Cancer Genome
+Consortium]](https://www.nature.com/articles/ng.3499?proof=true&draft=journal)
+: Good article about running a DAC
+
+[[Data Sharing in the Post-Genomic World: The Experience of the
+International Cancer Genome Consortium (ICGC) Data Access Compliance
+Office
+(DACO)]](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1002549)
+: A good reference on creating a tiered access data repository.
+
+[[DUO
+overview]](https://docs.google.com/presentation/d/1B4jsqnZIqwxLjL8Y1q41kYFNN8BsmmiEC6I9_WZZPt0/edit#slide=id.p)
+from GA4GH Basel presentation
+
+[[Data Use & Researcher
+Identities]](https://www.ga4gh.org/work_stream/data-use-researcher-identities-duri-2/)
+Effort to standardize and automate the granting of access based on data
+use and researcher identity patterns.
+
+[[DCP User
+stories]](https://docs.google.com/document/d/1uYdNEnBjl_tMJdGSh8f3VFDd0aw_hhhs2egyvkIfOeg/edit)
+(note controlled access user stories missing)
+
+[[NIST-800-53]](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-53Ar4.pdf)
+FedRAMP
+
+[[Fusillade
+RFC]](https://docs.google.com/document/d/1qD4bgnO684GtcLBErxYNZ5gvMaQPF8DsIdUBua5ZDBs/edit#heading=h.x01v6l5tv57z)
+describes the architecture for access control as implemented in the DCP
+
+### **Alternatives** 
+
+[[Near-term strategy for Managed Access in the
+DCP]](https://docs.google.com/document/d/1g_20xRCshmr5gmgr0OY64wW34r0DYxVRFpo6MhlXWkM/edit?pli=1#heading=h.hld7rftpusci).
+This approach uses a pre-existing external archive (data store) that
+already supports controlled access data. Identifiable data would be
+stored there with the derived non-identifiable data (expression data)
+being stored in the DCP.
```

Comment:
https://github.com/HumanCellAtlas/dcp-community/pull/100 has been created to represent the alternative, thanks